src/java.desktop/unix/classes/sun/awt/X11/XToolkit.java

```diff
@@ -1233,11 +1233,6 @@ public boolean getLockingKeyState(int key) {
 
     @Override
     public  Clipboard getSystemClipboard() {
-        @SuppressWarnings("removal")
-        SecurityManager security = System.getSecurityManager();
-        if (security != null) {
-            security.checkPermission(AWTPermissions.ACCESS_CLIPBOARD_PERMISSION);
```

Comment:
`import java.util.Hashtable;`
`import sun.awt.AWTPermissions;`

can be removed

Reply:
ok